common/src/client/apiClient.ts

```diff
@@ -61,23 +68,14 @@ const abortRun = async (client: HttpClient, conf: ApiClientConfig, runId: string
   }
 };
 
-const checkCloudCompatibility = async (client: HttpClient, conf: ApiClientConfig): Promise<void> => {
```

Comment:
Note: this endpoint is useless since its creation according to the backend history. Even if it has a long long saga of being migrated from a system to another 😅